app/code/community/Sendsmaily/Sync/Model/Rss.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/**
+ * Sendsmaily Sync
+ * Export Magento newsletter subscribers to Sendsmaily
+ * Copyright (C) 2010 Sendsmaily
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class Sendsmaily_Sync_Model_Rss extends Mage_Rss_Model_Rss
+{
+
+    /**
+     * Add header to Smaily RSS feed
+     *
+     * @param array $data Header elements
+     * @return Sendsmaily_Sync_Model_Rss Smaily RSS-object
+     */
+    public function _addHeader($data = array())
```

Comment:
Shouldn't methods starting with an underscore have to be either `protected` or `private` in Magento 1.x?

Reply:
I am extending the [Magento RSS model](https://github.com/OpenMage/magento-lts/blob/1.9.4.x/app/code/core/Mage/Rss/Model/Rss.php). They use `_functionName` syntax there.

Reply:
OK.